test/client_test.go

```diff
@@ -0,0 +1,28 @@
+package test
+
+import (
+	"testing"
+
+	mongodbatlas "go.mongodb.org/atlas/mongodbatlasv2"
+)
+
+func Test_Client_Config(t *testing.T) {
+	apiKey := "test"
+	apiSecret := "test"
+	host := "https://cloud.mongodb.com"
+
+	sdk, err := mongodbatlas.NewClient(
+		mongodbatlas.UseDigestAuth(apiKey, apiSecret),
+		mongodbatlas.UseBaseURL(host),
+		mongodbatlas.UseDebug(false))
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	serverURL := sdk.GetConfig().Servers[0].URL
+
+	if serverURL != host {
+		t.Errorf("Expected %s, got %s", host, serverURL)
+	}
+}
```

Comment:
let's do this an example
```suggestion
func ExampleNewClien() {
	apiKey := "test"
	apiSecret := "test"
	host := "https://cloud.mongodb.com"

	sdk, err := mongodbatlas.NewClient(
		mongodbatlas.UseDigestAuth(apiKey, apiSecret),
		mongodbatlas.UseBaseURL(host),
		mongodbatlas.UseDebug(false))

	if err != nil {
		panic(err)
	}

	serverURL := sdk.GetConfig().Servers[0].URL
	fmt.Println(serverURL)
	// Output:
	// https://cloud.mongodb.com
}
```